<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MalzemeCikisFormu</class>
 <widget class="QDialog" name="MalzemeCikisFormu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>481</width>
    <height>372</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ürün Çıkışı</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>451</width>
     <height>341</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="0" alignment="Qt::AlignmentFlag::AlignRight">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>Çıkış tarihi :</string>
      </property>
     </widget>
    </item>
    <item row="12" column="0" colspan="2" alignment="Qt::AlignmentFlag::AlignBottom">
     <widget class="QTableWidget" name="tableWidget_cikislar">
      <row>
       <property name="text">
        <string>New Row</string>
       </property>
      </row>
      <column>
       <property name="text">
        <string>Ürün Adı</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Ürün Miktarı</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Çıkış Tarih</string>
       </property>
      </column>
     </widget>
    </item>
    <item row="1" column="0" alignment="Qt::AlignmentFlag::AlignRight">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Ürün Miktarı :</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QPushButton" name="pushButtonStokDurumu2">
      <property name="text">
       <string>Stok Durumunu Göster</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1" alignment="Qt::AlignmentFlag::AlignLeft">
     <widget class="QLineEdit" name="lineEdit_7"/>
    </item>
    <item row="0" column="0" alignment="Qt::AlignmentFlag::AlignRight">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Ürün Adı :</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1" alignment="Qt::AlignmentFlag::AlignLeft">
     <widget class="QLineEdit" name="lineEdit_5"/>
    </item>
    <item row="1" column="1" alignment="Qt::AlignmentFlag::AlignLeft">
     <widget class="QLineEdit" name="lineEdit_6"/>
    </item>
    <item row="4" column="0">
     <widget class="QPushButton" name="pushButtonKaydet2">
      <property name="text">
       <string>Kaydet</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QPushButton" name="pushButtonTutanakli">
      <property name="text">
       <string>Tutanaklı Malzeme Çıkışı</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
